content_scripts/link_hints.coffee

```diff
@@ -805,7 +824,7 @@ LocalHints =
       element = visibleElement.element
 
       # Check middle of element first, as this is perhaps most likely to return true.
-      elementFromMiddlePoint = document.elementFromPoint(rect.left + (rect.width * 0.5), rect.top + (rect.height * 0.5))
+      elementFromMiddlePoint = LocalHints.getElementFromPoint(rect.left + (rect.width * 0.5), rect.top + (rect.height * 0.5))
       if elementFromMiddlePoint && (element.contains(elementFromMiddlePoint) or elementFromMiddlePoint.contains(element))
```

Comment:
Why check `elementFromMiddlePoint.contains(element)`? Is it for those with a CSS style of `pointer-events: none;` ?

Reply:
I seem to recall this was because sometimes `elementFromMiddlePoint` would be a parent to the clickable element, and I would get more consistent results when including this check. When testing out now, though, I can't seem to find any difference with or without it, so I guess it was some strange edge cases that might no longer be around.

Reply:
I find an opposite case: on https://material.angular.io/components/button-toggle/overview , the sidebar has overflow:hidden, so if I remove this call, those invisible list items will be not hinted.